README.md

```diff
@@ -99,13 +99,13 @@ var DropzoneDemo = React.createClass({
     },
 
     onOpenClick: function () {
-      this.refs.dropzone.open();
+      this.dropzone.open();
     },
 
     render: function () {
         return (
             <div>
-                <Dropzone ref="dropzone" onDrop={this.onDrop}>
+                <Dropzone ref={(el) => (this.dropzone = el)} onDrop={this.onDrop}>
```

Comment:
Let's switch to `ref={(node) => { this.dropzone = node; }}` since in your example it will implicitely return.